app/Resources/translations/messages.nl_NL.xliff

```diff
@@ -51,13 +62,19 @@
       </trans-unit>
       <trans-unit id="0ba164e8499593f2b376e03b4fb3483a41696c7f" resname="ss.form.ss_remote_vet_second_factor.cancel">
         <source>ss.form.ss_remote_vet_second_factor.cancel</source>
-        <target state="new">ss.form.ss_remote_vet_second_factor.cancel</target>
-        <jms:reference-file line="35">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetSecondFactorType.php</jms:reference-file>
+        <target state="new">Annuleren</target>
+        <jms:reference-file line="36">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetSecondFactorType.php</jms:reference-file>
+        <jms:reference-file line="60">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetValidationType.php</jms:reference-file>
       </trans-unit>
       <trans-unit id="5cfc619c2ed7de12fed04b48e45e9803de43c899" resname="ss.form.ss_remote_vet_second_factor.remote_vet">
         <source>ss.form.ss_remote_vet_second_factor.remote_vet</source>
         <target>Valideer identiteit</target>
-        <jms:reference-file line="31">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetSecondFactorType.php</jms:reference-file>
+        <jms:reference-file line="32">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Form/Type/RemoteVetSecondFactorType.php</jms:reference-file>
+      </trans-unit>
+      <trans-unit id="697ef084878c215ff97da5960df3bdc549cee9e7" resname="ss.form.ss_remote_vet_second_factor.validate">
+        <source>ss.form.ss_remote_vet_second_factor.validate</source>
+        <target>Bevestig</target>
```

Comment:
See feedback in English translation file, please update the Dutch translations accordingly.

Reply:
Done, except for form translations.